sos/plugins/kubernetes.py

```diff
@@ -96,6 +96,17 @@ def setup(self):
         self.add_cmd_output([
             "%s get %s" % (self.kube_cmd, res) for res in global_resources
         ])
+
+        # Get detailed node information
+        nodes = self.exec_cmd("%s get nodes" % self.kube_cmd)
```

Comment:
This command will be executed twice (via `add_cmd_output` few lines above as well). We can `collect_cmd_output` here and remove `nodes` from the `global_resources`, but that would leave `global_resources` incomplete..? Or some other solution to prevent twice call? (I am ok with the current implementation).

Reply:
Oh, good catch. We don't currently use `global_resources` anywhere else in the Plugin, so we're good to use `collect_cmd_output()` for nodes and remove that from the list.

I'll update shortly.